src/makeFileCopy.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+function getDirectoryPath(filePath) {
+  const splitedPath = filePath.split('/');
+
+  const directoryPath = splitedPath.slice(0, splitedPath.length - 1);
+
+  return directoryPath.join('/');
+}
+
+function makeFileCopy(fileToCopy, newFilePath) {
+  const fileToCopyPath = path.join(__dirname, fileToCopy);
+  const pathOfCopiedFile = path.join(__dirname, newFilePath);
+
+  if (getDirectoryPath(fileToCopyPath) === getDirectoryPath(pathOfCopiedFile)) {
+    return;
+  }
+
+  const fileData = fs.readFileSync(fileToCopyPath, 'utf-8');
+
+  fs.writeFileSync(pathOfCopiedFile, fileData);
```

Comment:
If file is not in text format, it will fail. Also u can use copyFileSync;

https://nodejs.org/api/fs.html#fscopyfilesyncsrc-dest-mode